frontend/src/components/DropDown/index.jsx

```diff
@@ -133,13 +133,14 @@ const AppDropDown = (selectedClass, setSelectedClass) => {
       console.log(res.errMsg);
     } else {
       const data = res.data;
-      // assume the field called classes
```

Comment:
AppDropDown should take in props as object {selectedClass, setSelectedClass}

Reply:
Might also need onSelect inside the object: {onSelect, selectedClass, setSelectedClass}

Reply:
sorry, I forgot to make that update to the new pull request. Will do